.phan/config.php

```diff
@@ -1,66 +1,74 @@
 <?php
 
 return [
-	"backward_compatibility_checks" => true,
-	// The docs on quick_mode at
-	// https://github.com/etsy/phan/wiki/Incrementally-Strengthening-Analysis
-	// don't seem reasonable. They claim that quick_mode=true add more errors.
-	// This is false on the assumption that that's a typo.
-	// It doesn't seem to have any effect on LORIS's codebase anyways.
-	"quick_mode" => false,
-	"analyze_signature_compatibility" => true,
-	// FIXME: allow_missing_properties should be false, but there's
-	// too many other things to fix first.
-	"allow_missing_properties" => true,
-	"null_casts_as_any_type" => false,
-	"scalar_implicit_cast" => false,
-	"ignore_undeclared_variables_in_global_scope" => false,
-	"suppress_issue_types" => [
+    "backward_compatibility_checks" => true,
+    // The docs on quick_mode at
+    // https://github.com/phan/phan/wiki/Incrementally-Strengthening-Analysis
+    // don't seem reasonable. They claim that quick_mode=true add more errors.
+    // This is false on the assumption that that's a typo.
+    // It doesn't seem to have any effect on LORIS's codebase anyways.
+    "quick_mode" => false,
+    "analyze_signature_compatibility" => true,
+    // The default severity level for phan is 5. After removing all the
+    // suppressed rules, we should consider reducing this value to detect more
+    // suspicious code.
+    "minimum_severity" => 5,
+    // FIXME: allow_missing_properties should be false, but there's
+    // too many other things to fix first.
+    "allow_missing_properties" => true,
+    "null_casts_as_any_type" => false,
+    "scalar_implicit_cast" => false,
+    // The line below is required to prevent PhanUndeclaredVariable problems in
+    // the bvl_feedback ajax scripts. They all require() a file in this directory
+    // that declares the $feedbackThread variable. Phan doesn't know about this.
+    //
+    // Toggling the below rule to true is the suggested fix by the phan devs.
+    // see https://github.com/phan/phan/issues/1650.
+    //
+    // When the module is refactored, this line should be deleted.
+    "ignore_undeclared_variables_in_global_scope" => true,
+    // FIXME: We should add this.
+    "dead_code_detection" => false,
+    // FIXME: We should add this. Note that dead_code_detection also covers
+    // unused_variable_detection. It might be a good idea to enable this first.
+    "unused_variable_detection" => false,
+    "suppress_issue_types" => [
         "PhanTypeInvalidDimOffset",
         "PhanTypePossiblyInvalidDimOffset",
-		"PhanUndeclaredMethod",
+        "PhanUndeclaredMethod",
         "PhanTypeMismatchDimFetch",
-		"PhanTypeMismatchReturn",
-		"PhanUndeclaredClassMethod",
-		"PhanTypeMismatchArgument",
-		"PhanTypeMismatchProperty",
+        "PhanTypeMismatchReturn",
+        "PhanUndeclaredVariableDim",
```

Comment:
When I look at this with `?w=1`, this seems to be accidentally sneaking PhanUndeclaredVariableDim into the list of ignores.